### ISSUE TITLE
VolView doesn't display any images.

### ISSUE THREAD

**Issue by olegyuryev** [Kitware/VolView#330]
Good afternoon! 
VolView doesn't display any images (rest of the app seems ok, but all render windows are blank). Restarting chrome doesn't help, all render windows are blank. Examples of my dicom files in attachment.

[DX.zip](https://github.com/Kitware/VolView/files/11479852/DX.zip)

**Comment by PaulHax** [comment]
Hello!  Thanks for the report.

On my Linux Chrome, when I drag and drop each of the .dcm files in your zip they show up.  

Are the render windows white?  

Does this example work?
https://kitware.github.io/vtk-js/examples/ZipHttpReader.html

Any WebGL website work ?
https://webglsamples.org/blob/blob.html

**Comment by olegyuryev** [comment]
Thanks for the reply. I'm sorry, I didn't specify that I open the program on Windows 10 Chrome. I'm using a dark theme and I see black windows and gray bars.

![image](https://github.com/Kitware/VolView/assets/53606528/8f82d1f0-3a11-4cc8-8404-aa543d4fc03d)

**Comment by PaulHax** [comment]
That banded image in lower left is not right.  We should fix.  

If you load each DICOM file individually, they work.

**Comment by floryst** [comment]
I'm aware of this issue with DX images. We will be working to fix this soon. Thanks for letting us know!

**Comment by olegyuryev** [comment]
By the way, loading files individually helped. But this is not convenient, I plan to use VolView in DICOM server Orthanc. Orthanc loads a whole study with several files at once.
Thanks for the help, we are waiting for a fix. Have a nice day!

**Comment by danthila** [comment]
Does this affect if I run an older tag locally ?

**Comment by aylward** [comment]
Thanks Forrest - adding this as a high priority for the 4.2 release is the right next step!

**Comment by floryst** [comment]
I think you have a different issue. Do the [vtk.js examples](https://kitware.github.io/vtk-js/examples/) have the same issue that you experience with a local build?

**Comment by The-D-lab** [comment]
> I think you have a different issue. Do the [vtk.js examples](https://kitware.github.io/vtk-js/examples/) have the same issue that you experience with a local build?

Yes i was able to run the examples locally on a different/new vue project

**Comment by udda89** [comment]
> I think you have a different issue. Do the [vtk.js examples](https://kitware.github.io/vtk-js/examples/) have the same issue that you experience with a local build?

Yes  , I was able to run the samples on a new vue project

**Comment by aylward** [comment]
Need a 2D image flag for DX, PDF, ...
